keras/utils/multi_gpu_utils.py

```diff
@@ -4,17 +4,21 @@
 from __future__ import division
 from __future__ import print_function
 
+import tensorflow as tf
 from ..layers.merge import concatenate
 from .. import backend as K
 from ..layers.core import Lambda
 from ..engine.training import Model
 from ..models import clone_model
 from ..utils.generic_utils import to_list
 
+def _is_tf_1():
+    return tf.__version__.startswith('1.')
 
 def _get_available_devices():
```

Comment:
Maybe we could put this in `tensorflow_backend`, and have `_get_available_gpus` rely on it?